pep-0631.rst

```diff
@@ -58,7 +58,9 @@ Every element SHOULD be an `entry <#entries>`_.
     ]
 
 Any element MAY be an inline table to support possible future functionality.
-Backends MUST silently ignore these until the specification is extended.
+Backends MUST error when encountering these until the specification is extended.
+Other parsers SHOULD do the same, though they MAY consider the entire enclosing
```

Comment:
This is still not right. The ideal situation is that every parser would mark the relevant field as dynamic (which basically means "ask the backend" — the backend, in turn, will raise an error unless it's coded against a newer version of the spec).

Saying that parsers SHOULD raise an exception defeats the entire purpose of the backwards compatibility component, so this should read:

> Backends MUST error when encountering these until the specification is extended; other parsers SHOULD consider the entire enclosing field dynamic.

Possibly a version that makes the rationale a bit clearer would be:

> Backends MUST error when encountering these unless the specification is extended; other parsers — particularly those not intended to be listed in a project's `build-system.requires` — SHOULD consider the entire enclosing field dynamic.